quickstarts/microsoft.azurestackhci/vm-simple-windows/azuredeploy.parameters.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "vm-sampledeploy"
+        },
+        "location": {
+            "value": "eastus"
+        },
+        "adminUsername": {
+            "value": "admin-quickstart"
+        },
+        "adminPassword": {
+            "value": "[newGuid()]"
+        },
+        "vCPUCount": {
+            "value": 2
+        },
+        "memoryGB": {
+            "value": 4
+        },
+        "imageName": {
+            "value": "GEN-UNIQUE" 
```

Comment:
same question hear about `imageName` as the other PR

Reply:
imageName will be unique to the customer environment